boefjes/tests/test_app.py

```diff
@@ -22,10 +22,13 @@ def test_one_process(manager: SchedulerWorkerManager, item_handler: MockHandler)
 
     patched_tasks = manager.scheduler_client.get_all_patched_tasks()
 
-    assert len(patched_tasks) == 3
-    assert patched_tasks[0] == ("70da7d4f-f41f-4940-901b-d98a92e9014b", "completed")
+    assert len(patched_tasks) == 6
```

Comment:
@Donnype would you mind taking a look if I'm making the correct assumptions within these tests.

Reply:
@jpbruinsslot These patches seem logical right? All of them go from running to either completed or failed :+1: